index.js

```diff
@@ -314,12 +356,12 @@ class LedgerBridgeKeyring extends EventEmitter {
     } else {
       accounts = this._getAccountsLegacy(from, to)
     }
+    //this._sendMessage({ action: 'ledger-close-bridge' }, () => this.forgetDevice())
```

Comment:
Need to remove this, as it disconnects Ledger Live after retrieving the account listing, leading to the user needing to connect again